### PR TITLE
Update NetPlayServer.cpp

### PR DESCRIPTION
**Pull request by jefflongo**
Make Netplay pings refresh more frequently to monitor ping better.

### CODE REVIEW

**Comment by JMC47** [comment]
@dolphin-emu-bot rebuild

looks good to me.

**Comment by jefflongo** [comment]
Thanks @JMC47 , by the way it's YL good to see ya.

**Comment by JMC47** [comment]
Hah, making the leap from user to developer I see!  I don't think there's any reason not to merge this.

File: Source/Core/Core/NetPlayServer.cpp [hunk 1]
```diff
@@ -48,7 +48,7 @@ void NetPlayServer::ThreadFunc()
 	while (m_do_loop)
 	{
 		// update pings every so many seconds
-		if ((m_ping_timer.GetTimeElapsed() > (10 * 1000)) || m_update_pings)
+		if ((m_ping_timer.GetTimeElapsed() > (1000)) || m_update_pings)
```
Comment (lioncash) [inline Source/Core/Core/NetPlayServer.cpp]: You can remove the parentheses around the 1000

Comment (jefflongo) [inline Source/Core/Core/NetPlayServer.cpp]: Fixed.

**Comment by JosJuice** [comment]
The two commits could be squashed into one. Looks good other than that.

**Comment by johnwchadwick** [comment]
In case you're newer to git, here is one way you could merge or edit commits:
```
git rebase -i HEAD~2
```

Also, if you ever just want to edit your last commit, a very convenient line is this:
```
git commit -a --amend
```
(but I recommend checking `git diff` to ensure that does not pull in unwanted changes.)

After doing either of these, you will need to force push, as normally git will not allow you to push anything that is 'destructive' (removes commits from the tree.)
```
git push -f origin patch-1
```

**Comment by jefflongo** [comment]
Sorry I'm new to this.. where do I enter these commands and how do I set them up?

**Comment by Buddybenj** [comment]
Download GitHub for Windows here: https://windows.github.com/
Download GitHub for Mac here: https://mac.github.com/

Then, you will be able to enter Git commands in Git Shell.

**Comment by jefflongo** [comment]
I have Github, I'm just confused on what I do next.  Entering git rebase -i HEAD~2 brings up a text editor where I assume I squash.  What do I need to do to squash these 2 commits?  Is this affecting my local directory or this PR?

**Comment by JosJuice** [comment]
The text editor displays a list of your commits, with the word `pick` in front of them. Change `pick` to `squash` for the second commit, then save and exit, and it will be combined with the first commit. This will update your local repository. Use `git push -f` as described before in order to also update your GitHub repository (and by extension, the PR).

**Comment by jefflongo** [comment]
It's now locally updated but running the git push -f origin patch-1 command I get this..


Thanks so much for your help.

**Comment by Buddybenj** [comment]
Try `git push -f`

**Comment by jefflongo** [comment]
I've just run that it says everything up to date.  Is there a reason it's not updating here?

**Comment by Buddybenj** [comment]
If it says everything up to date, that means you have no local changes. Are you sure you pressed "Save" after squashing the commit in your text editor?

Wait, on the picture you posted you were on the master branch, you should be on the patch-1 branch. Try `git checkout patch-1`

**Comment by jefflongo** [comment]
That was it @Buddybenj , thanks so much!  All should be ready to go now.

**Comment by jefflongo** [comment]
@jmc47 can you approve this?

**Comment by Sonicadvance1** [comment]
@dolphin-emu-bot rebuild

**Comment by JMC47** [comment]
I can't approve the build beyond saying that it seems like a good idea.

**Pull request merged**